examples/extract-auth-token.py

```diff
@@ -4,12 +4,43 @@
 import libflagship.logincache
 import json
 import sys
+import platform
+import os
+import getopt
 
-if len(sys.argv) != 2:
-    print(f"usage: {sys.argv[0]} <path/to/login.json>")
-    exit(1)
+def print_login(filename):
+    data = open(filename).read()
+    jsonj = json.loads(libflagship.logincache.decrypt(data))
+    print(jsonj["data"]["auth_token"])
 
-data = open(sys.argv[1]).read()
+def main():
+    userdir = os.getlogin()
+    useros = platform.system()
 
-json = json.loads(libflagship.logincache.decrypt(data))
-print(json["data"]["auth_token"])
+    darfileloc = f'/Users/{userdir}/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json'
```

Comment:
can you test this?

```python
    darfileloc = os.path.expanduser('~/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json')
```

Then we don't need to make assumptions about where homedir is located (which is what we're really trying to do)